### ISSUE TITLE
Change the tomato color to make it accessible

### ISSUE THREAD

**Issue by obulat** [creativecommons/chooser#283]
## Problem

The Help links in the Chooser are of color `tomato`. When it is on the `light-gray` background, the contrast does not meet accessibility standards. 
![Tomato links on the light gray background](https://user-images.githubusercontent.com/15233243/103862297-aaaf2c00-5073-11eb-8f46-825811f10b32.png)
![Contrast check](https://user-images.githubusercontent.com/15233243/103862303-ad118600-5073-11eb-8dad-3787e7fd5dcd.png)

## Description

This is why the color has been [adjusted](https://github.com/creativecommons/vocabulary/pull/842) in the Vocabulary repo. 

This color needs to be adjusted in the Chooser repo here: https://github.com/creativecommons/chooser/blob/151ea88041fa87c9d88664f25270ed602d14e136/src/components/HelpSection.vue#L241 The new color should be #C74200 .

## Additional context

This is a temporary solution until the color changes are released in Vocabulary. After that, we will use the Vocabulary `tomato` scss variable.

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by rohan-goel** [comment]
Hey, I would like to work on this issue.

**Comment by obulat** [comment]
Sure, @rohan-goel , go ahead!
